metadata.json

```diff
@@ -2,8 +2,8 @@
   "name": "puppetlabs-reboot",
   "version": "0.1.10-SNAPSHOT",
   "author": "puppetlabs",
-  "summary": "Adds a type and provider for managing Windows system reboots.",
-  "license": "Apache License, Version 2.0",
+  "summary": "Adds a type and provider for managing system reboots.",
+  "license": "Apache-2.0",
```

Comment:
We seem to be inconsistent about the license, e.g. mysql uses [Apache-2.0](https://github.com/puppetlabs/puppetlabs-mysql/blob/master/metadata.json#L6), apache uses [Apache 2.0](https://github.com/puppetlabs/puppetlabs-mysql/blob/master/metadata.json#L6), and acl uses [Apache License, Version 2.0](https://github.com/puppetlabs/puppetlabs-acl/blob/master/metadata.json#L6). I defer to @cyberious about best practices.